cypress/component/ProgressBar.cy.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+
+import { ProgressBar } from "../../src/ProgressBar/ProgressBar";
+
+
+const testProps = {
+  "aria-label": "Loading…",
+  "data-id": "progressBar",
+  "aria-valuenow": "50"
+};
+const progressBar = '[data-id="progressBar"]';
+
+describe("ProgressBar", () => {
+  it("renders  with value and accesibility label", () => {
+    cy.mount(<ProgressBar isIndeterminate {...testProps} />);
+    cy.get('[data-id="progressBar"]')
+      .should("have.attr", "aria-label", testProps["aria-label"])
+      .should("have.attr", "aria-valuenow").and("to.have.string", testProps["aria-valuenow"])
+  });
+
+  it("renders class with custom class name", () => {
+    cy.mount(
+      <ProgressBar className="cypress-test" value={50} {...testProps} />
+    );
+
+    cy.get(progressBar)
+      .should("have.attr", "class")
+      .and("to.have.string", "ProgressBar")
+      .and("to.have.string", "cypress-test");
+  });
+
+  it("forwards ref", () => {
+    const ref = React.createRef<HTMLDivElement>();
```

Comment:
Just a note: These 2 tests could probably be reused for most of the components.

Reply:
I've copied it from ProgressCircle, so I'll leave it separately for now